components/description/description.module.scss

```diff
@@ -0,0 +1,55 @@
+$colors: #456c3d, #669f54, #7ec16a, #b9e7ac, #ecedf0;
+
+.squareContainer li {
+  width: 3vmin;
+  height: 3vmin;
+  border-radius: 5px;
+  margin: 15px 0;
+  transform: rotate(-90deg);
+}
+
+@for $i from 1 through length($colors) {
+  .squareContainer li:nth-of-type(#{$i}) {
+    background: darken(nth($colors, $i), 0%);
+    border: 5px solid darken(nth($colors, $i), 5%);
+    box-shadow: 0 0 5px 1px lighten(nth($colors, $i), 25%);
+  }
+}
+
+.container {
+  width: 100%;
+  min-height: 45vh;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 4vmin;
+  font-weight: 300;
+  font-family: Assistant;
```

Comment:
No need, it was already declared in the global file: https://github.com/urish/pull-request-community/blob/master/styles/globals.scss